### ISSUE TITLE
00 -- TERRY CALHOUN

### ISSUE THREAD

**Issue by drumslayert** [TIY-Durham/2015-FALL-FEE#3]
- [x] **GitHub-ing to Know You**
  - [x] [Create a GitHub account](https://github.com/signup)
  - [x] [Edit your GitHub profile](https://github.com/settings/profile) to provide:
  - [x] A picture, so we can recognize your face.
  - [x] Your full name, so we can recognize your name.
  - [x] A valid public email address, so we can contact you.
  - [x] Find and star [the class project](https://github.com/TIY-Durham/2015-FALL-FEE)
  - [x] Read [the Home page on the class wiki](https://github.com/TIY-Durham/2015-FALL-FEE/wiki)
- [x] **You've Got Issues**
  - [x] [Create a new _WIP Issue_](https://github.com/TIY-Durham/2015-FALL-FEE/issues/new)
  - [x] Name it `00 -- YOUR NAME` (where `YOUR NAME` is _your_ name)
  - [x] Make this checklist the description
  - [x] Start checking off boxes... :astonished:
- [x] **Send all my GitHub to you...**
  - [x] Learn all you can about `git` and GitHub!
    - Treehouse has [a Git Basics course](http://teamtreehouse.com/library/git-basics)
  - CodeSchool has [a `git` path](https://www.codeschool.com/paths/git)
    - [x] Complete: [Try Git](http://try.github.com)
  - [x] GitHub has a couple of Guides:
    - [x] [Hello World](https://guides.github.com/activities/hello-world/)
    - [x] [Forking Projects](https://guides.github.com/activities/forking/)
  - [x] Add a link to your `Hello-World` and `Fork-and-Spoon` repos as a comment on your _WIP Issue_
- [x] **Command Line Training**
  - [x] Read: http://eddywashere.com/blog/an-intro-to-the-os-x-command-line/
  - [x] Complete: https://www.codecademy.com/courses/learn-the-command-line/
  - [x] **BEAST MODE** http://cli.learncodethehardway.org/book/
- [x] **INSTALL ALL THE THINGS!**
  - [x] [Install Homebrew](http://brew.sh) on your Mac
  - [x] Open `Terminal.app`; Don't Panic.
  - [x] [Install Homebrew and the `brew` command](http://brew.sh/#install)
  - [x] Read [the Homebrew Wiki](https://github.com/Homebrew/homebrew/tree/master/share/doc/homebrew#readme) to learn more about Homebrew
  - [x] Using the `brew` command, install the following packages:
    - [x] `zsh`
  - [x] `git`
  - [x] `hub`
  - [x] `node`
  - [x] Add the output of the following commands as comments on your _WIP Issue_:
    - [x] `which zsh && zsh --version`
    - [x] `which git && git --version`
    - [x] `which hub && hub --version`
    - [x] `which node && node --version`
  - [x] Install the following (bonus if you use `brew`):
    - [x] [Google Chrome](http://google.com/chrome)
    - [x] [Atom](http://atom.io)
    - [x] [Dash](http://kapeli.com/dash)
    - [x] [Slack](http://www.slack.com)
    - [x] [SourceTree](https://www.sourcetreeapp.com/)
- [x] **Prezto Change-oh!**
  - [x] Find and _fork_ the Prezto project on GitHub
  - [x] Clone _your fork_ of Prezto into your home directory
  - [x] Follow the installation instructions in the `README.md` file
- [x] **Take a Moment to Reflect**
  - [x] Answered question 1 in a comment on your WIP Issue.
  - [x] Answered question 2 in a comment on your WIP Issue.

**Comment by drumslayert** [comment]
The link in "You've Got Issues: Create an Issue in the class repo (this one)"  is broken,  so I'm just assuming this might be the same repo?  As you can see I pasted the "tasks" in here,  so please kindly correct me if I'm wrong and let me know where to find the right one. Thank you

**Comment by kellymurray** [comment]
Thanks @drumslayert - the link has been fixed!

**Comment by drumslayert** [comment]
hello world : https://github.com/drumslayert/hello-world.git

Spoon-knife :  https://github.com/drumslayert/Spoon-Knife.git

**Comment by drumslayert** [comment]
zsh- /bin/zsh
zsh 5.0.5 (x86_64-apple-darwin14.0)

node-  /usr/local/bin/node
v0.12.7

git -  /usr/bin/git
git version 2.3.2 (Apple Git-55)

hub-  /usr/local/bin/hub
git version 2.3.2 (Apple Git-55)
hub version 2.2.1

**Comment by drumslayert** [comment]
question 1:
I have been out of school for over 20 years, so at the very least I am predicting to encounter quite an adventure. A much welcomed adventure that I've been looking forward to. From the pre-work I anticipate encountering that some aspects of Javascript will be difficult to grasp at first. The better I get at "thinking in code" the more the headaches should sub-side.  
  In my past, when I've stepped out to learn something new that was challenging like this, I would surrounded myself with people that could help me accomplish my goal. Experienced people that I could mimic and learn from. 
  I plan on asking a lot of questions and seeking help until I have the answers. I'm very laid back and easy-going. I'm the calm one In most situations and environments. I get along and work well with others, so I expect to do the same here. I hope to make a lot of new great friends here, in the staff as well as my classmates. I hope to work together as a team helping each other, encouraging and learning from one another. I expect nothing less, than all of us to come out on the other side of this cohort as your super nerds.

**Comment by drumslayert** [comment]
question 2:
 I'd be lying if I didn't tell you that I enjoyed all of it to some degree. It's hard for me to pick a favorite. It was overcoming the challenges in all of it that felt great. I probably spent the majority of any spare time I could find these last few weeks in treehouse and codeacademy classes playing around with html, css and javascript. 
 The part I liked the least was not being able to finish some of my online tracs with JavaScript. Because of some severe time restraints, I didn't get to go as far with it as I did html and css. Im confident I'll be seeing plenty of it very soon though. Also trying to remember all of the different commands and syntax between all the different languages has not been my fave.
